package.json

```diff
@@ -20,7 +20,12 @@
   "dependencies": {
     "@aws-amplify/adapter-nextjs": "^1.2.19",
     "@aws-amplify/backend": "^1.2.0",
+    "@aws-sdk/client-appsync": "^3.650.0",
+    "@aws-sdk/client-dynamodb": "^3.654.0",
```

Comment:
The tests should operate as their own app with a separate package.json so that we don't pollute the production code with unnecessary test code or in this case dependencies.

Reply:
updated